apps/events/models.py

```diff
@@ -10,9 +10,11 @@
 from apps.chat.models import ChatMessage, ChatMessageType
 from apps.events.actions import end_conversation, log, summarize_conversation
 from apps.experiments.models import Experiment, ExperimentSession
+from apps.experiments.models import Experiment, ExperimentSession, SessionStatus
 from apps.utils.models import BaseModel
 
 ACTION_FUNCTIONS = {"log": log, "end_conversation": end_conversation, "summarize": summarize_conversation}
+STATUSES_FOR_COMPLETE_CHATS = [SessionStatus.PENDING_REVIEW, SessionStatus.COMPLETE, SessionStatus.UNKNOWN]
```

Comment:
In [chat/tasks.py](https://github.com/dimagi/open-chat-studio/blob/9976a4d49bdb4925711de57417a543332790a628/apps/chat/tasks.py#L22) there is also a `STATUSES_FOR_COMPLETE_CHATS` Just wondering if we should read from there (or a more central place)?

Reply:
I think we might remove it from there if we are going to remove the no-activity-ping functionality, which is why I copied it here.